paddle/fluid/eager/backward.cc

```diff
@@ -728,6 +729,8 @@ std::vector<paddle::experimental::Tensor> RunBackward(
                        "got edges size is: %d, grad_output size is: %d",
                        metas.size(), grad_output_tensors.size()));
 
+    std::deque<egr::GradNodeBase*> accumulation_node_queue;
```

Comment:
Bad name, this should be a reordered queue, and give some comments for why we need this.